BinaryCounter.py

```diff
@@ -0,0 +1,19 @@
+from abc import ABC
+
+from Counter import Counter
+
+
+class BinaryCounter(Counter, ABC):
+    def __init__(self, name, mass, price):

```

Comment:
You did type annotations for your base class ```Device```, but it's better to add them for every class, as technically I'm able to do things like ```binary_counter.price = "not an integer!"```, thinking everything's correct, because ```__init__``` for ```BinaryCounter``` is technically not type annotated.